src/components/Percentagebar.js

```diff
@@ -0,0 +1,22 @@
+// completion is just a number value as completion="25"
+function PercentageBar({ completion }) {
+  let barColor = completion === 100 ? "bg-green-500" : "bg-blue-500";
+  let barwidth = (completion / 100) * 140;
+
+  return (
+    <div className="w-[150px] h-[30px] flex flex-wrap items-center">
+      <p className="w-[80%] h-4 text-xs font-bold text-gray-500">
+        {`${completion}%`}
```

Comment:
This is fine to keep, but another way to write it (maybe more simple) is:
`{completion}%`

Reply:
thanks for pointing it out, changed it